File: Sirius/CodeTest/2_Condition_Statement/no_6.py
h,m= map(int, input().split())
recipe = int(input())
re_m=m+recipe
if(re_m<60):
    print(h,re_m)
else:
    ad_h=re_m//60
    re_m%=60
    h+=ad_h
    if(h>=24): h-=24
    print(h,re_m)

File: Sirius/CodeTest/2_Condition_Statement/no_6.cpp
#include <iostream>
using namespace std;
int main()
{
    int h,m, recipe,re_m;
    cin >> h >> m >> recipe;
    re_m= m+recipe;
    if(re_m < 60) cout << h << ' ' << re_m;
    else
    {
        int ad_h=re_m/60;
        re_m%=60;
        h+=ad_h;
        if(h>=24) h-=24;
        cout << h << ' ' << re_m;
    }
    return 0;
}